### ISSUE TITLE
Status autobonus

### ISSUE THREAD

**Issue by admkakaroto** [rathena/rathena#1970]
rAthena Hash: 
Client: 2015-11-04
Mode: Renewal.

Hello, the combo of these 2 items, when generating the autobonus are causing a confusion in the attributes of the character. To reproduce the problem is simple.

Team these 2 items: 5074:18827

After attacking any monster until the transformation happens, however you need to have some attribute points distributed for the problem to occur, if you have 1 attribute attribute the change does not affect that attribute of the character, see the image: 

Img 1: http://i.imgsafe.org/63e1b2354c.png
img 2: http://i.imgsafe.org/63e2432b1a.png

Full Stats: http://i.imgsafe.org/63e321836f.png

**Comment by aleos89** [comment]
So you're saying that the bonuses from the item go away when you're in transform mode?

**Comment by admkakaroto** [comment]
The odd bonus is valid during the period that the player is online, after relog in the account the status returns to normal.

**Comment by jezznar** [comment]
I guess this video shows that there shouldn't be any negative bonuses at all?
https://www.youtube.com/watch?v=-VQocQqK8y0
during my test using Hash: 65fa13089 and 2015-11-04, the negative bonus stats come in the moment I transformed into Valkyrie. It the negative bonus stats don't come off even if I died but it goes off after I `@refresh`  or relogin.
I can confirm that autobonus does not have anything to do with this as I tried taking off that part of the script and it happens.
